src/Tests/AutologoutTestSessionCleanupOnLoginTest.php

```diff
@@ -11,7 +11,7 @@
  *
  * @description Ensure that the autologout module cleans up stale sessions at login
  *
- * @group autologout
+ * @group Autologout
```

Comment:
Please update the group name in AutologoutAjaxTestCaseTest.php tests file as well